### ISSUE TITLE
What parameters were used for the pre-trained model?

### ISSUE THREAD

**Issue by MWPainter** [bearpaw/pytorch-pose#58]
Hi, 

I've been trying to run this code and been encountering similar issues as other below, such as validation accuracy being far worse than training accuracy and so on.

So far, I have updated the batch norm momentum to 0.5, which seems to fix the validation vs training accuracy problem. The other change that I have made is reducing the learning to 1e-4, as also recommended in discussions for other issues.

However, after 60 epochs or so, both training and validation accuracies are below still 0.5, whereas the quoted results for any number of stacks is above 0.5 by 20 epochs. 

**Would you be able to provide the exact commend typed into the terminal to train your "hg_s8_b1" model, as I believe this would help others be able reproduce your training runs?**

(For example, I would like to use a slightly modified part of this network as part of a larger system in my work, so it would be massively useful if I could actually get training to work).

thanks in advance,
Michael

**Comment by bearpaw** [comment]
Unfortunately, this issue may be caused by the updated version of PyTorch and has been reported several times. For stable training, please use PyTorch v0.2.0
